ct/cmd/install.go

```diff
@@ -31,13 +31,17 @@ func newInstallCmd() *cobra.Command {
 		Use:   "install",
 		Short: "Install and test a chart",
 		Long: heredoc.Doc(`
-			Run 'helm install' and ' helm test' on
+			Run 'helm install', 'helm test', and optionally 'helm upgrade' on
```

Comment:
Do you want to pull ee32efab79ca923f7baac9e2ed7ac7772bc33ea0 out of this PR now that this is split from #103? It may make this easier to review.

Reply:
Hmm... I suppose I could point this PR to my own branch, but that's on my fork so I don't think that'll work. Otherwise I don't think there's a way to prevent the diff from #103 showing while also keeping the new tests passing.

Reply:
Most everything is in https://github.com/helm/chart-testing/pull/120/commits/6c91a169655f6d13fedd9db5f9101758645c231c (with a small change to use recently updated example hostpath configuration in https://github.com/helm/chart-testing/pull/120/commits/bcbb4a50b8fe69b6b3b4cad94c12578d4e242ba5). 😄

Reply:
Never mind, I now see this depends on #103. You noted it in the initial description 😄 Perfect